server/routes/api/chart.py

```diff
@@ -74,9 +74,8 @@ def get_statsvars_need_all_dates(chart_list):
     """
```

Comment:
this function isn't required anymore since we need to keep all dates. could you add a note to remove this?

Reply:
also the get*date_for_stat_vars

Reply:
Added a todo to remove them